_notices/rgn0013.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 13 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "RAPIDS switching to CalVer in v0.20"
```

Comment:
How about "RAPIDS switching to CalVer in v0.20 (now v21.06.00)"? Just want to not promote the old 0.20 name any longer.

Reply:
Done